packages/babel-plugin-remove-graphql-queries/package.json

```diff
@@ -8,6 +8,10 @@
     "directory": "packages/babel-plugin-remove-graphql-queries"
   },
   "homepage": "https://github.com/gatsbyjs/gatsby/tree/master/packages/babel-plugin-remove-graphql-queries#readme",
+  "dependencies": {
+    "@babel/runtime": "^7.14.0",
+    "gatsby-core-utils": "^2.8.0-next.1"
```

Comment:
The `dependencies` field is specified twice now so this one will be ignored

Reply:
Thanks, i'll fix it. I probably did a bad merge here